docs/documentation/quartz-3.x/tutorial/more-about-jobs.md

```diff
@@ -54,6 +54,11 @@ class before calling its `Execute(..)` method. One of the ramifications of this
 have a no-arguement constructor. Another ramification is that it does not make sense to have data-fields defined 
 on the job class - as their values would not be preserved between job executions.
 
+:::tip Dependency Injection
+If using quartz with a dependency injection framework, your constructor can pull in service dependencies just
+like a controller in an ASP.Net MVC.
```

Comment:
> Quartz might not be ideal in ASP.NET environment due to life-cycles/recycles so I usually give advice to have separate process/windows services/cloud worker for this - it can work in web context, but I usually get too many reports about problems than happiness.

That's a section I want to explore in a dedicated area.